src/platforms/android/configuration/integrations/navigation.mdx

```diff
@@ -0,0 +1,151 @@
+---
+title: Navigation
+caseStyle: camelCase
+supportLevel: production
+sdk: sentry.java.android.navigation
+description: "Learn more about the Sentry Navigation integration for the Android SDK."
+categories:
+  - mobile
+---
+
+The `sentry-android-navigation` library provides [AndroidX Navigation](https://developer.android.com/jetpack/androidx/releases/navigation) support for Sentry using the [SentryNavigationListener](https://github.com/getsentry/sentry-java/blob/524ee49b212c3f2eead20960c3c6825ccdbf8007/sentry-android-navigation/src/main/java/io/sentry/android/navigation/SentryNavigationListener.kt). The source can be found [on GitHub](https://github.com/getsentry/sentry-java/tree/main/sentry-android-navigation/src/main/java/io/sentry/android/navigation).
+
+On this page, we get you up and running with Sentry's Navigation Integration, so that it will automatically add a breadcrumb and start an idle transaction for each navigation event.
```

Comment:
Similar review on this line https://github.com/getsentry/sentry-docs/pull/5234/files